### DISCUSSION TITLE
Any support for waveshare 2.9inch eink display?

### DISCUSSION THREAD

**Discussion by raxp122** [guysoft/FullPageOS#566] [Q&A]
I want to display a website in my 2.9inch eink display and set a refresh rate for the screen of about 5 seconds and refresh the website in the same interval. Is there any easy solution or guide?

**Comment by guysoft** [comment]
Not something that I know of. You can't refresh pages separately on FullPageDashboard. So you would have to develop a way to refresh webpages somehow.
